src/main/java/gift/service/product/ProductService.java

```diff
@@ -15,30 +17,33 @@ public ProductService(ProductReposiotory productReposiotory) {
     }
 
     public List<ProductDto> getProducts() {
-        return List.copyOf(productReposiotory.selectAllProducts()
-                            .stream()
-                            .map(ProductDto::from)
-                            .toList()
-                            );
+        return productReposiotory.findAll()
+            .stream()
+            .map(ProductDto::from)
+            .collect(Collectors.toList());
```

Comment:
이렇게도 사용해 볼 수 있습니다 ㅎㅎ
```suggestion
            .toList();
```